src/message.js

```diff
@@ -108,7 +112,9 @@ Globalize.prototype.messageFormatter = function( path ) {
  */
 Globalize.formatMessage =
 Globalize.prototype.formatMessage = function( path /* , variables */ ) {
-	return this.messageFormatter( path ).apply( {}, slice.call( arguments, 1 ) );
+	return ( arguments[ 1 ] && typeof arguments[ 1 ] === "boolean" ) ?
+		this.messageFormatter( path, arguments[ 1 ] ).apply( {}, slice.call( arguments, 2 ) ) :
+		this.messageFormatter( path ).apply( {}, slice.call( arguments, 1 ) );
```

Comment:
I double checked this with FF Firebug and it proved to be multiple arguments, i.e. in my use case 'arguments' constitutes the plain array object with length corresponding to number of parameters of 'formatMessage' call.
BTW, I probably ought to add this additional parameter explicitly just as it has been done for 'messageFormatter'?